### PR TITLE
User story001

### PR DESCRIPTION
**Pull request by Suna898789**
Please, if you don't mind review my PR.
Thank you

### CODE REVIEW

File: src/test/resources/features/logout.feature
Comment (Shohruh0312) [inline src/test/resources/features/logout.feature:1]: looks good

**Review by Shohruh0312 — APPROVED** [review]
looks good,good to continue

**Comment by Shohruh0312** [comment]
good job

**Review by Kelyxa — APPROVED** [review]
All good!

**Review by Shohruh0312 — COMMENTED** [review]
good

**Pull request merged**